infra-lifecycle-wg/charter/charter.md

```diff
@@ -0,0 +1,84 @@
+
+### Infrastructure Tool Orchestration
+
+The growing need for GitOps-style deployments drives demand for orchestrating declarative cloud infrastructure using Infrastructure as Code Tools. These orchestrators manage cloud resources either using native APIs or IaC tools. They offer flexibility across cloud providers and tooling preferences. Tools like TACoS, Crossplane, and AWS Controller for Kubernetes (ACK) exemplify this powerful approach, simplifying cloud infrastructure management.
+
+## Examples of known tools aimed to manage the lifecycle of infrastructure:
+
+Without any claim to comprehensiveness:
+
+* Ansible
+* AWS CloudFormation & AWS CDK
+* Bicep
+* Crossplane
+* Metal³
+* OpenTofu
+* Pulumi
+* Radius
+* Terraform
+* Tinkerbell
+
+## Alignment with TAG App Delivery Charter
+
+The Infrastructure Lifecycle working group tackles the underlying infrastructure abstractions that serve as essential building blocks for platform products, an area addressed by the Platform Working Group. Application delivery is often coupled with infrastructure, think of databases, message queues and runtime environments. Originally, the Platform Working Group focused on clarifying the coordination between application development and infrastructure delivery. While this remains important, the group's focus has evolved to consider platforms as complete products. This broader perspective necessitates a renewed emphasis on the underlying infrastructure abstractions that provide the essential building blocks for these platform offerings, such as disaster recovery, infrastructure lifecycle state management, testability and more. The Infrastructure Lifecycle Working Group will address such topics in this TAG charter.
+
+## Alignment with TAG Runtime Charter
+
+We partner with the Runtime TAG to ensure that cloud-native runtime environments provide the fundamental workload execution abstractions and mechanisms needed to integrate with infrastructure lifecycle management tools. This collaboration focuses on guaranteeing that both runtime and infrastructure layers work cohesively to deliver applications effectively.
+
+## Working mode/expected outcome
+
+The Infrastructure Lifecycle Working Group will achieve its goals through regular meetings ([Zoom](https://zoom-lfx.platform.linuxfoundation.org/meeting/96148400770?password=767d45df-c7cf-4400-9239-e789115cc85e&invite=true)), open communication channels ([#wg-infrastructure-lifecycle](https://cloud-native.slack.com/archives/C06USDTN683) on Slack), and collaboration with other TAGs. This will result in a standardised approach for managing the lifecycle of infrastructure in a cloud-native setting, develop a set of comprehensive best practices, and publish valuable resources for the cloud-native community around infrastructure lifecycle management
```

Comment:
Hi, could you specify what is the success criteria of the WG?  I'd like to understand when we should continue or discontinue the WG.

Reply:
Hello @linsun, I'm not sure if this is something that we intend to define in the charter itself as I haven't seen examples of this in other WG charters. Do you have any examples? I happy to bring to up for discussion in our next WG meeting.

Reply:
@linsun Actually, @TheFoxAtWork left some excellent comments about the stated goals of the WG in the charter that I think we could clarify and address, which would also address your question. Please let me know what you think.

https://docs.google.com/document/d/1eK7m13F8SHuL2ji0U__5S_dlD2UBX0POquj6AtBWkoc/edit?disco=AAABOxARaAk

Reply:
Thank you, I will follow up in the google doc :)

Reply:
We have significantly narrowed the scope of this working group to focus on a single deliverable: _develop a framework for managing the lifecycle of infrastructure in public, private, or hybrid cloud environments based on cloud-native principles. This framework will guide end-users in managing infrastructure to ensure it is secure, resilient, manageable, sustainable, and observable._